app/views/reports/regions/_patient_breakdown_charts.html.erb

```diff
@@ -74,14 +74,16 @@
       <div class="pt-20px px-20px">
         <div class="d-flex flex-1 mb-8px">
           <h3 class="mb-0px mr-8px">
-            Assigned patients breakdown
+            Hypertension patients
           </h3>
           <%= render "definition_tooltip",
                      definitions: { "Patients under care" => t("patients_under_care_copy"),
-                                    "Lost to follow-up" => t("lost_to_follow_up_copy.reports_card_subtitle") } %>
+                                    "Lost to follow-up" => t("lost_to_follow_up_copy.reports_card_subtitle"),
+                                    "Transferred out" => t("transferred_out_patients_copy")} %>
         </div>
-        <p class="mb-0px c-grey-dark mr-lg-12px"><%= number_with_delimiter(@chart_data[:patient_breakdown][:total_patients]) %>
-          hypertension patients assigned to <%= @region.name %> as of <%= @chart_data[:ltfu_trend].last_value(:period_info)["bp_control_end_date"] %></p>
+        <p class="mb-0px c-grey-dark mr-lg-12px">
+          Hypertension patients who have died, been lost to follow-up, and transferred out
```

Comment:
I would say "or transferred out" instead of "and". Otherwise it sounds like they did all three, which would be impressive.

Reply:
Good call, thanks @kpethtel !